tic_tac_toe/board.py

```diff
@@ -0,0 +1,48 @@
+from tic_tac_toe.views import BoardDisplay
+
+
+class Board:
+    def __init__(self):
+        self.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    def __getitem__(self, index):
+        return self.board[index]
+
+    def turn(self):
+        view = BoardDisplay()
+        token = self.current_player()
+
+        print("Please enter 1-9.")
+
+        view.display_board(self.board)
+
+        user_input = int(input()) - 1
```

Comment:
What happens if the player enters a character that cannot be converted to an int?